charts/stable/nextcloud/questions.yaml

```diff
@@ -179,10 +179,11 @@ questions:
                     - variable: max_allowed_resolution
                       label: Max Allowed Resolution
                       schema:
-                        type: int
+                        type: string
+                        valid_chars: '^[0-9]{2}(\.[0-9]{1})?$'
```

Comment:
It actually can be bigger float eg 222.22 
Which is what upstream Nextcloud sets on their aio. Let’s update the 2 to 3 and 1 to 2

thanks